scripts/batch-score.coffee

```diff
@@ -0,0 +1,121 @@
+# Description:
+#   Script for batch wise score.
+#
+# Configuration:
+#   INFO_SPREADSHEET_URL
+#
+# Commands:
+#   hubot score bxx
+#
+# Author:
+#   aseem09
+
+module.exports = (robot) ->
+
+  scorefield = () ->
+    Field = robot.brain.get("scorefield") or {}
+    robot.brain.set("scorefield",Field)
+    Field
+
+  stringLength = (str) ->
+    string=String(str)
+    return string.split('').length
+
+  # makes all the elements in the array equal in width by padding at right
+  padright = (array) ->
+    padding=0
```

Comment:
I know we don't have any styling guide ... yet. But let's follow some of the standard styles till then?
Some guidelines:
1. There should be spaces around `=`
2. Any line should at most be of 80 chars length
3. `then` should come on the next line of `if` (same for `else`)
4. Avoid global variables

Reply:
Okay